src/helpers/Sign/crypto/index.js

```diff
@@ -0,0 +1,474 @@
+/**
```

Comment:
is there any pre-existing library we can use here instead of having the code explicitly implemented?